src/components/integrated/OakLessonVideoTranscript/OakLessonVideoTranscript.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactNode, useState } from "react";
+
+import { OakBox, OakFlex } from "@/components/base";
+import { OakCollapsibleContent, OakTertiaryButton } from "@/components/ui";
+
+type OakLessonVideoTranscriptProps = {
+  /**
+   * The transcript content
+   */
+  children: ReactNode;
+  /**
+   * The id of the collapsible content element. This is used to link the button to the content
+   */
+  id: string;
+  /**
+   * A control to toggle the video to display sign language
+   */
+  signLanguageControl?: ReactNode;
+};
+
+/**
+ * Display a togglable video transcript with a slot to display a sign language control
+ */
+export const OakLessonVideoTranscript = ({
+  children,
+  id,
+  signLanguageControl = null,
+}: OakLessonVideoTranscriptProps) => {
+  const [showTranscript, setShowTranscript] = useState(false);
+
+  return (
+    <>
+      <OakFlex
+        $flexDirection={["column-reverse", "row"]}
```

Comment:
The sign-language control will be displayed above the transcript toggle at the mobile breakpoint as in the designs.